src/components/global/ContactFormModal.jsx

```diff
@@ -0,0 +1,138 @@
+// React Imports
+import { useRef } from 'react';
+// emailjs Imports
+import emailjs from '@emailjs/browser';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Grid from '@mui/material/Grid';
+import TextField from '@mui/material/TextField';
+// Other Library Imports
+import { PropTypes } from 'prop-types';
+
+// TODO: Change title from "Contact Us" to "Volunteer" if using that button?
+
+const ContactFormModal = ({ showContactFormModal, setShowContactFormModal }) => {
+  const form = useRef();
+
+  const handleClose = () => {
+    setShowContactFormModal(false);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    emailjs.sendForm('service_co2agxf', 'template_20pnwni', form.current, 'AkBl59Ya3226OfPyQ').then(
+      (result) => {
+        console.log(result.text);
+      },
+      (error) => {
+        console.log(error.text);
+      }
+    );
+    e.target.reset();
```

Comment:
need to add a `setShowContactFormModal(false)` in here too, or form stays after submitting.